website/src/categories/components/avatar/usage.md

```diff
@@ -57,6 +57,9 @@ In the context of contact lists, avatars serve as visual cues that swiftly conne
 ### Intrusive Design
 Don’t let avatars distract from essential content. Ensure they enhance, rather than hinder, usability.
 
+### Avatar actions
+Avatar link variant shouldn't used to trigger actions. It is intended solely for linking to another page. The users typically expect clickable avatars to navigate to profile pages or related content. By reserving the avatar link variant for page navigation, we maintain a clear and consistent user experience, ensuring that interactions with avatars are predictable and intuitive.
```

Comment:
That depends on the case. Usually, we use 'Users'. You can see that in the 1st text of this page, use 'users'. That is because we refer to the collective of the future users.